layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/PortalNavigationPortlet_en.properties

```
@@ -24,3 +24,4 @@ UISiteManagement.label.editPortalProp=Edit Site Configuration
 UISiteManagement.label.deletePortal=Delete
 UISiteManagement.msg.delete-global-portal=Can't delete global portal site
 UISiteManagement.label.noSitesAvailable=No editable site navigation.
+UISiteManagement.label.siteNavigation=Site navigation.
```

```suggestion
siteNavigation.button.tooltip.label=Site navigation
```
move the labels to a separated file SiteNavigationPortlet_en.properties